kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -4625,6 +4630,15 @@ messages:
             if Send(Send(Send(SYS,@GetSurvivalRoomMaintenance),
                      @FindRoomByGuild,#oGuild=poGuild),@GetAllowJoins)
             {
+               lInventory = send(self,@GetHolderPassive);
```

Comment:
Instead of lInventory and GetHolderPassive, use for i in plPassive here. GetHolderPassive just returns plPassive for Users.